test/integration/authentication-oidc-test.js

```diff
@@ -332,8 +378,8 @@ describe('Authentication API (OIDC)', () => {
               })
           })
 
-          it('should return a 403', () => {
-            expect(response).to.have.property('status', 403)
+          it('should return a 401', () => {
```

Comment:
Not terribly important, but as per the comment above the test, I still think 403 is the correct status. How hard would it be to fix?